bns/getKoboData.js

```diff
@@ -1,45 +1,45 @@
+import { http } from '@openfn/language-googlesheets';
 //== Job to be used for fetching data from Kobo on repeated, timer basis  ==//
 // This can be run on-demand at any time by clicking "run" //
+getValues(
+  '1s7K3kxzm5AlpwiALattyc7D9_aIyqWmo2ubcQIUlqlY',
+  'sheetsList of Kobo forms!A:L',
+  state => {
+    const [headers, ...values] = state.data.values;
+
+    const filteredSheetsData = values.filter(item => {
+      if (
+        item.includes('TRUE') &&
+        item.includes('bns_survey', 'nrgt_current')
+      ) {
+        return item;
+      }
+    });
+
+    state.sheetsData = filteredSheetsData.map(item => {
```

Comment:
Hiya @hunterachieng i think we can improve this logic here 

```
const mapHeaderToValue = (value) => {
  headers.reduce((obj, header) => {
    obj[header] = value[headers.indexOf(header)];
    return obj;
  }, {});
};

state.sheetsData = values
  .filter(
    (item) =>
      item.includes("TRUE") && item.includes("bns_survey", "nrgt_current")
  )
  .map((item) => mapHeaderToValue(item));

```
I am not sure if `mapHeaderToValue()` is the right function name but the idea is to decouple the mapping of key value pair for each column.